Solutions/PrismaCloudCompute/Data Connectors/dataConnectorPoller.json

```diff
@@ -0,0 +1,48 @@
+{
+    "type": "Microsoft.SecurityInsights/dataConnectors",
+    "apiVersion": "2022-10-01-preview",
+    "name": "apiRequest",
+    "kind": "RestApiPoller",
+    "properties": {
+        "connectorDefinitionName": "connectorDefinitionName",
```

Comment:
The value should be the connectordefinition id here

Reply:
done